Library/Homebrew/extend/ENV/std.rb

```diff
@@ -149,11 +149,6 @@ def gcc
     end
   end
 
-  def llvm
-    super
-    set_cpu_cflags
-  end
```

Comment:
I didn't realize the `llvm` method in the superclass is auto-generated from the now removed `:llvm` symbol and is unlikely to be called directly via its name. And I found no trace of it being used anywhere, so I guess my point about `compat/` is moot.